angular.js

```diff
@@ -32,5 +32,28 @@ module.exports = {
             Overrides to Angular extra recommended rules:
             https://github.com/angular-eslint/angular-eslint/blob/master/packages/eslint-plugin/src/configs/recommended--extra.json
         */
+
+        /*
+            Overrides to Angular rules outside of the recommended configuration:
+        */
+
+        '@angular-eslint/component-max-inline-declarations': 'error',
+        '@angular-eslint/component-selector': 'error',
+        '@angular-eslint/contextual-decorator': 'error',
+        '@angular-eslint/directive-selector': 'error',
+        '@angular-eslint/no-attribute-decorator': 'error',
+        '@angular-eslint/no-forward-ref': 'error',
+        '@angular-eslint/no-input-prefix': 'error',
+        '@angular-eslint/no-lifecycle-call': 'error',
+        '@angular-eslint/no-pipe-impure': 'error',
+        '@angular-eslint/no-queries-metadata-property': 'error',
+        '@angular-eslint/pipe-prefix': 'error',
+        '@angular-eslint/prefer-on-push-component-change-detection': 'error',
+        '@angular-eslint/prefer-output-readonly': 'error',
+        '@angular-eslint/relative-url-prefix': 'error',
+        '@angular-eslint/sort-ngmodule-metadata-arrays': 'error',
```

Comment:
[@angular-eslint/sort-ngmodule-metadata-arrays](https://github.com/angular-eslint/angular-eslint/blob/master/packages/eslint-plugin/docs/rules/sort-ngmodule-metadata-arrays.md)

Reply:
Similarly to sort-vars, sort-keys, sort-imports and sort-type-union-intersection-members, sorting array members alphabetically is not a generally expected practice. Although, some like myself may do so anyway.

**Proposal**
Leave the rule disabled.

Reply:
@mure this thread on @angular-eslint/sort-ngmodule-metadata-arrays is awaiting your vote

Reply:
@TrevorKarjanis this is ready to close.